engine/dbc/sc_spell_info.cpp

```diff
@@ -1435,6 +1435,14 @@ std::string spell_info::to_str( const dbc_t& dbc, const spell_data_t* spell, int
     s << ( int ) spell -> max_range() << " yards" << std::endl;
   }
 
+  if ( spell -> max_targets() != 0 )
+  {
+    fmt::print( s, "Max Targets      : {}{}{}\n",
+      spell -> max_targets() < 0 ? "Unlimited(" : "",
```

Comment:
Might make sense to call this ´Unknown´ instead of Unlimited, from the spell query dump we can see some values of at least -1 and -6, meaning Blizzard might be using negative values to denote some thing server-sidey.

Reply:
Yeah, so `-1` is def unlimited from what I've seen. So I decided to do:
* `Unlimited(-1)` if n == -1
* `Unknown(n)` otherwise if n < 0

Reply:
Yep, that's good.